frontend/src/app/hierarchy.service.ts

```diff
@@ -47,16 +47,23 @@ export class HierarchyService {
         return this.http.post<Hierarchy>(url, hierarchy);
     }
 
-    getFakeSensitivityAnalysis(nodeNames: string[]): Observable<SensitivityAnalysisReport>{
+    deleteHierarchy(hierarchyId: string): Observable<string> {
+        const url = this.root + '/hierarchy/' + hierarchyId;
+        return this.http.delete<string>(url);
+
+        // Select another hierarchy or show welcome screen
```

Comment:
We probably wouldn't select another hierarchy here. I think we would do it in the component after a successful delete.